src/theme/ThemeProvider.js

```diff
@@ -30,6 +35,9 @@ const theme = {
       opacity: 0.5,
     },
   },
+  opacity: {
+    disabled: 0.5,
+  },
```

Comment:
very noiceeeee (this broke my local site for the longest time after pulling latest changes, and i couldn't figure out why lmao)